datalake-spark3/src/test/scala/bio/ferlab/datalake/spark3/utils/ClassGeneratorSpec.scala

```diff
@@ -138,12 +138,6 @@ case class TestClassOutput(`a`: Option[String] = None,
       Seq(TestInput(d = Seq())).toDF,
       LocalDateTime.of(1900, 1, 1, 0, 0, 0))
 
-    ClassGenerator.writeCLassFile(
```

Comment:
Je suis d'accord que ce call ne sert à rien dans la réussite du test, mais le test `"class generator" should "create folder if not exists"` ne vérifie pas le résultat de l'écriture. On pourrait le retirer du test de Timestamp, mais écrire un test qui vérifie que le fichier écrit par `writeCLassFile` est valide?

Reply:
Oui le test est pertinent en effet. Par contre on devrait deleter le fichier du repo et ajouter une entrée dans le .gitignore